src/main/java/com/emeq/counter/service/FoodService.java

```diff
@@ -0,0 +1,70 @@
+package com.emeq.counter.service;
+
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import com.emeq.counter.dto.FoodDTO;
+import com.emeq.counter.exception.RecordNotFoundException;
+import com.emeq.counter.model.Food;
+import com.emeq.counter.repository.FoodRepository;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FoodService {
+
+    @Autowired
+    private FoodRepository foodRepository;
+
+    private static final String EXCEPTION_STRING = "Invalid Food Id : ";
+
+    public ResponseEntity<List<FoodDTO>> index() {
+        Iterable<Food> foods = foodRepository.findAll();
+
+        List<FoodDTO> result = StreamSupport.stream(foods.spliterator(), false).map(Food::convert2DTO)
```

Comment:
if we are going for multi line chaining, does it make more sense to keep each call in its own lines?

Something like this:

```java
List<FoodDTO> result = StreamSupport
  .stream(foods.spliterator(), false)
  .map(Food::convert2DTO)
  .collect(Collectors.toList());
```

Reply:
I changed here like you say. But I used two tab (8 spaces) for indentation.